plugins/smart_proxy_dhcp_infoblox/control

```diff
@@ -10,6 +10,6 @@ XS-Ruby-Versions: all
 Package: ruby-smart-proxy-dhcp-infoblox
 Architecture: all
 XB-Ruby-Versions: ${ruby:Versions}
-Depends: ${shlibs:Depends}, ${misc:Depends}, ruby | ruby-interpreter, foreman-proxy (>= 1.17.0~rc1), ruby-infoblox (>= 2.0.4), ruby-infoblox (<< 3)
+Depends: ${shlibs:Depends}, ${misc:Depends}, ruby | ruby-interpreter, foreman-proxy (>= 1.17.0~rc1), ruby-infoblox
```

Comment:
Well as the infoblox gem is archived, it is very unlikely for it to reach a version 4, but well, it might be adopted some day and then this limitation can make sense, so i've added it.